b6func.py

```diff
@@ -301,9 +302,11 @@ def edi_resample(src, w, h, edi=None, kernel='spline16', a1=None, a2=None,
                             etype=None, pscrn=None, device=None)
 
 
-def scaled_grain(clip: vs.VideoNode, grain_w: int, grain_h: int, var: float,
+def scaled_grain(clip: vs.VideoNode, var: float = 1,
+                 grain_h: Optional[int] = None, grain_w: Optional[int] = None,
                  static: bool = True, adaptive: bool = False,
-                 luma_scaling: int = 12, kernel: str = 'bicubic') -> vs.VideoNode:
+                 luma_scaling: int = 12, kernel: str = 'bicubic',
+                 b: float = 1/3, c: float = 1/3, taps: int = 3) -> vs.VideoNode:
```

Comment:
Any particular reason for 1/3? If possible I'd like to make it `b=0` `c=0.5` to keep it consistent with zimg/VS defaults. Actually thinking about it a bit more it might be better to make them optional with a default of `None` like `fvsfunc.Resize`. Thoughts?

Reply:
I'm fine with changing that. Not sure if a fallback is really necessary here, especially since I don't know what `fvsfunc.Resize` defaults back to (I couldn't find it after a quick check).

Reply:
Yeah not thinking of a fallback value specifically, but I wonder if passing through `None` for these three params by default here will just use the zimg defaults downstream. If that's the case then any changes to the defaults downstream would automatically change here too.

I'd like to test that out.

Reply:
I don't have time to test it right now, but you can easily confirm this by comparing absolute differences between a clip downscaled with the method presented here and one when normally downscaled using `fvsfunc.Resize`. Should be a piece of cake.

Reply:
Yep. I'll test it out when I get a chance so I may end up changing it down the line. Not a show stopper right now.